docs/integration/laravel.md

```diff
@@ -0,0 +1,107 @@
+---
+category: integration
+---
+
+# AsyncAws for Laravel
+
+The AsyncAws integrations for Laravel are similar to the integrations for the
+official AWS SDK. The sections below describe how you can install and configure
+Cache, Filesystem, Mail and Queue.
+
+## Illuminate Cache
```

Comment:
So we must be clear on cache can work for redis or DynamoDb. Did we test out redials cache driver?

Reply:
The RedisStore has nothing to do with AWS. It just connects to a Redis server, it does not matter where it is hosted.

Reply:
I added a comment that it only supports DynamoDB

Reply:
But 99% if all Laravel apps uses redis as cache driver

Reply:
Then this package will be used by only 1% of the users =)

I also agree that Redis is a far better cache provider than DynamoDb. 

If we would add a RedisStore in this package it will be 100% exactly the same as `Illuminate\Cache\RedisStore`.